frontstage/templates/register/register-share-survey-enter-your-details.html

```diff
@@ -0,0 +1,182 @@
+
+{% extends "layouts/_block_content.html" %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/lists/_macro.njk" import onsList %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% from "components/password/_macro.njk" import onsPassword %}
+{% from "components/fieldset/_macro.njk" import onsFieldset %}
+{% from "components/label/_macro.njk" import onsLabel %}
+{% set page_title = "Share survey enter account details" %}
+
+{% block main %}
+    {% if errors|length > 0 %}
+        {% if errors|length == 1 %}
+            {% set errorTitle = 'There is 1 error on this page' %}
+        {% elif errors|length > 1 %}
+            {% set errorTitle = 'There are ' ~ errors|length ~ ' errors on this page' %}
+        {% endif %}
+        {% call
+            onsPanel({
+                "type": "error",
+                "classes": "u-mb-s",    
+                "title":  errorTitle
+            })
+        %}
+            <p>These <strong>must be corrected</strong> to continue.</p>
```

Comment:
This should be in the `{% if errors|length == 1 %}` block, eg if errors|length == 1 then use **This** rather than **These**

Reply:
Not sure I get the comment @mfcrocker. currently its if error is greater than 0 go in if error is one use this title else user check if its greater than 1 use this.

Reply:
Sorry its duplicate - looks like if I provide my response in files section its not reflected in conversation.
Not sure I get the comment @mfcrocker. currently its
`if error is greater than 0`
`if error is one`
use this title
`else if error is more than one`
user another title.